From 42b389822dc220cac56b5bd51fa600643a3ae1aa Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Tue, 7 Feb 2023 09:35:25 +0100
Subject: [PATCH 406/529] ARM: configs: enable sd modulator on
 fragment-02-multiv7_addons

Enable SD modulator on multiv7 addons fragment to support DFSDM
IIO use cases.

Signed-off-by: Olivier Moysan <olivier.moysan@st.com>
Change-Id: I61eca300d04666de2be3291d445f76f4f1f2eb35
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/289556
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Olivier MOYSAN <olivier.moysan@foss.st.com>
Reviewed-by: Olivier MOYSAN <olivier.moysan@foss.st.com>
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -57,6 +57,7 @@ CONFIG_RPMSG_VIRTIO=y
 CONFIG_SAMPLES=y
 CONFIG_SAMPLE_RPMSG_CLIENT=y
 CONFIG_SCHED_MC=y
+CONFIG_SD_ADC_MODULATOR=y
 CONFIG_SENSORS_IIO_HWMON=y
 CONFIG_SERIAL_NONSTANDARD=y
 CONFIG_SRAM=y
